src/main/java/com/snowflake/kafka/connector/internal/streaming/DirectTopicPartitionChannel.java

```diff
@@ -1110,13 +873,13 @@ public long getProcessedOffset() {
   @Override
   @VisibleForTesting
   public long getLatestConsumerOffset() {
-    return this.latestConsumerOffset.get();
+    return 0;
```

Comment:
ditto previous comment - lets do -1 or something to indicate its never used

Reply:
done